mi/dataset/parser/auv_common.py

```diff
@@ -127,7 +143,7 @@ def parse_file(self):
             line = line.strip()  # remove the line terminator
             line = line.replace('"', '')  # remove the quote characters from string fields
 
-            for message_id, field_count, compute_timestamp, particle_class in self._auv_message_map:
+            for message_id, field_count, _compute_timestamp, particle_class in self._auv_message_map:
```

Comment:
`_compute_timestamp` used here as a variable, and later as a function.

Where is `_compute_timestamp` defined?

Reply:
I wanted to differentiate the variable, compute_timestamp, in the for statement from the actual function compute_timestamp defined at the top.

_compute_timestamp is defined in the "for" statement as the other variables are in that statement.

Reply:
PEP-8 suggests the use of the leading underscore in two cases:

_single_leading_underscore: weak "internal use" indicator. E.g. from M import * does not import objects whose name starts with an underscore.

Use one leading underscore only for non-public methods and instance variables.

So, I suggest you just use something local, like `timestamp`.